inception-project/inception
# #1191 - Fix link to docker compose

**What's in the PR**
* Fix the link to docker compose yaml

**How to test manually**
* Build the doc locally, check the admin guide, click on link to docker compose file

**Automatic testing**

**Documentation**
